src/opamp.rs

```diff
@@ -370,46 +488,76 @@ macro_rules! opamps {
         ),*
     } => {
         paste::paste!{
-            $(impl <IntoInput, IntoOutput> IntoFollower <IntoInput, IntoOutput, $input> for Disabled
+            $(
+            impl <IntoInput, IntoOutput> IntoFollower <$opamp, IntoInput, IntoOutput, $input, $output> for Disabled<$opamp>
                 where
                     IntoInput: Into<$input>,
                     IntoOutput: Into<$output>,
             {
                 fn follower(
                     self,
                     input: IntoInput,
-                    output: Option<IntoOutput>,
-                ) -> Follower<$input> {
+                    output: IntoOutput,
+                ) -> Follower<$opamp, $input, $output> {
+                    let input = input.into();
+                    let output = output.into();
+                    unsafe {
+                        use crate::stm32::opamp::[<$opampreg _csr>]::OPAINTOEN_A;
+                        (*crate::stm32::OPAMP::ptr())
+                            .[<$opampreg _csr>]
+                            .write(|csr_w|
+                                csr_w
+                                    .vp_sel()
+                                    .$input_mask()
+                                    .vm_sel()
+                                    .output()
+                                    .opaintoen()
+                                    .variant(OPAINTOEN_A::OutputPin)
```

Comment:
Not sure if it would be worth it, but what if we had a trait with an associated const `OPAINTOEN_A` that was implemented for external and InternalOutput types?

Reply:
Take a look at the other change I made on 13d4631 and let me know what you think. I'll play with the trait idea.

Reply:
Taking that one step further, what if all follower, pga, etc defaulted to internal only? Then the user could enable the output builder style. That would probably compose well with optional inverting inputs etc.

Reply:
I thought about that but I think complexity will increase a lot (e.g. we would need to store various configurations of outputs attached to the struct). I don't think that makes sense given the limited surface area of the API. I think this is probably the 'correct' way to do it though.